pnpm-workspace.yaml

```diff
@@ -1,3 +1,3 @@
 packages:
-  - js
+  - packages/js-sdk
   - python
```

Comment:
Why is the JS package in `./packages/js-sdk` and the Python in `./python`?

Reply:
Because this PR isn't done yet, it's a draft

Reply:
Both in `packages` now